src/ResourceManager/Reservations/Commands.Reservations/Cmdlets/PatchReservation.cs

```diff
@@ -6,6 +6,12 @@
 using System.Management.Automation;
 using Newtonsoft.Json;
 using Microsoft.Azure.Management.Reservations;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
```

Comment:
@corquiri please add a changelog entry in your changelog for this change.  Also, there should be an automated test for this scenario.  Please add a  test or file a test debt issue to add it next sprint.